test/ComposerRequireCheckerTest/UsedSymbolsLocator/LocateUsedSymbolsFromASTRootsTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace ComposerRequireCheckerTest\UsedSymbolsLocator;
+
+use ArrayObject;
+use ComposerRequireChecker\UsedSymbolsLocator\LocateUsedSymbolsFromASTRoots;
+use PhpParser\Node\Name;
+use PhpParser\Node\Stmt\Class_;
+use PHPUnit\Framework\TestCase;
+
+class LocateUsedSymbolsFromASTRootsTest extends TestCase
```

Comment:
Please add `/** @covers \ComposerRequireChecker\UsedSymbolsLocator\LocateUsedSymbolsFromASTRoots */` to prevent coverage leaks